pkg/v1/providers/infrastructure-azure/v1.0.1/cconly/overlay.yaml

```diff
@@ -0,0 +1,61 @@
+#@ load("@ytt:overlay", "overlay")
```

Comment:
@vuil We've merged Azure v1.2.1 to main. We probably should rebase this branch off main and merge. Or merge all the Azure related changes in a separate PR. Thoughts?

Reply:
thanks, rebased and updated to 1.2.1